templates/default/rt/rtcategorylist.html

```diff
@@ -72,10 +72,10 @@ <H1>{$layout.pagetitle}</H1>
 			{if $category.lastticket}{$category.lastticket|date_format:"%Y/%m/%d %H:%M:%S"}{else}-{/if}
 		</TD>
 		<TD class="lms-ui-buttons text-right nobr">
-			<A HREF="?m=rtqueueview&amp;catid={$category.id}"><IMG SRC="img/view.gif" alt="{trans("Browse")}" title="{trans("Browse")}"></A>
-			<A class="delete-rtcategory" HREF="?m=rtcategorydel&id={$category.id}"><IMG src="img/delete.gif" alt="{trans("Delete")}" title="{trans("Delete")}"></A>
-			<A HREF="?m=rtcategoryedit&amp;id={$category.id}"><IMG SRC="img/edit.gif" alt="{trans("Edit")}" title="{trans("Edit")}"></A>
-			<A HREF="?m=rtcategoryinfo&amp;id={$category.id}"><IMG SRC="img/info.gif" alt="{trans("Info")}" title="{trans("Info")}"></A>
+			{button type="link" href="?m=rtqueueview&amp;catid={$category.id}" icon="view" tip="{trans("Browse")}"}
+			{button type="link" class="delete-rtcategory" href="?m=rtcategorydel&id={$category.id}" icon="delete" tip="{trans("Delete")}"}
+			{button type="link" href="?m=rtcategoryedit&amp;id={$category.id}" icon="edit" tip="{trans("Edit")}"}
```

Comment:
&amp;amp w href możemy zamieniać na & - to jakaś pozostałość po zgodności z HTML 4 STRICT.

Reply:
OK. Dzięki za naukę. Zapamiętam.

Reply:
Zaraz poprawię.